readme.md

```diff
@@ -120,7 +120,6 @@ As defined by Wikipedia a shim is a small library that transparently intercepts
 
 - [CanvasPanel](http://canvas-panel.netlify.com/) - React library to build IIIF Presentation 3 level viewing experiences including support for annotations.
 - [Chronoscope World](https://mprove.de/chronoscope/world.html) – A viewer for georeferenced IIIF maps. It is also a fast viewer for browsing IIIF books.
-- [Clover IIIF](https://samvera-labs.github.io/clover-iiif/) - IIIF Presentation API Manifest viewer handling Image, Sound, and Video canvases for React.js
```

Comment:
If we can, I think Clover IIIF still belongs under the IIIF Viewers heading. If we need to be specific about which components in Clover are _viewers_, we can say **Clover IIIF Viewer** and link directly to it at https://samvera-labs.github.io/clover-iiif/docs/viewer.

Reply:
Agree with @mathewjordan above.  Point directly at the `Viewer` component 👍